data_prep/introspector.py

```diff
@@ -48,13 +49,14 @@
 
 def _set_introspector_endpoints(endpoint):
   """Sets URLs for Fuzz Introspector endpoints to local or remote endpoints."""
-  global INTROSPECTOR_ENDPOINT, INTROSPECTOR_CFG, INTROSPECTOR_FUNCTION, INTROSPECTOR_SOURCE, INTROSPECTOR_XREF, INTROSPECTOR_TYPE, INTROSPECTOR_FUNC_SIG
+  global INTROSPECTOR_ENDPOINT, INTROSPECTOR_CFG, INTROSPECTOR_FUNCTION, INTROSPECTOR_SOURCE, INTROSPECTOR_XREF, INTROSPECTOR_TYPE, INTROSPECTOR_FUNC_SIG, INTROSPECTOR_ORACLE_1
```

Comment:
nit: Since we are changing this, would you please use line breaks to avoid overlong lines?
E.g., https://github.com/google/oss-fuzz-gen/pull/176/files#diff-2988c9b2abbf23c03fa8ed236ed0f62fb249e13ef31c1509b7fef0f6d78dd1beR51-R53